src/models/schema.test.ts

```diff
@@ -78,6 +80,30 @@ describe("Schema model methods", () => {
     });
     assert.equal(schema.fileName(), "test-topic.100123.v1.confluent.avsc");
   });
+
+  it("nextVersionDraftFileName() should return the correct file name", () => {
+    const schema = TEST_CCLOUD_SCHEMA.copy({
+      // @ts-expect-error: update dataclass so we don't have to add `T as Require<T>`
+      subject: "test-topicValue",
+      // @ts-expect-error: update dataclass so we don't have to add `T as Require<T>`
+      version: 1,
+    });
```

Comment:
This should work and keep TS happy, yeah?
```suggestion
    const schema = Schema.create({
      ...TEST_CCLOUD_SCHEMA,
      subject: "test-topicValue",
      version: 1,
    });
```